libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -616,16 +616,80 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
 
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
+            background: url("../../assets/data_source_normal_state.svg") no-repeat;
+        }
+
+        &.is-active::before {
+            background: url("../../assets/data_source_active_state.svg") no-repeat;
+        }
+
+        &.disabled::before {
+            background: url("../../assets/data_source_disabled_state.svg") no-repeat;
+        }
+
+        &::before {
+            top: 11px;
+            margin-top: 1.2px;
+
+            &.disabled {
+                color: $gd-color-disabled;
+            }
+
+            &.is-normal {
+                color: $gd-color-link;
+            }
+        }
+
         &:hover {
-            border-color: $gd-color-highlight;
+            z-index: 100;
+            box-shadow: 0 0 0 1px $gd-color-highlight;
+
+            &.disabled {
+                color: $gd-color-disabled;
+            }
+
+            &.is-normal {
+                color: $gd-color-link;
+            }
         }
     }
 
     .icon-model {
         border-radius: 0;
 
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
+            background: url("../../assets/model-normal-state.svg") no-repeat;
+        }
+
+        &::before {
+            top: 11px;
+            margin-top: 2px;
+
+            &.disabled {
+                color: $gd-color-disabled;
+            }
+
+            &.is-normal {
+                color: $gd-color-link;
+            }
```

Comment:
these lines are repeated
reuse?

Reply:
I created a function that we can reuse it thanks